src/ssort/_parsing.py

```diff
@@ -128,15 +128,21 @@ def split_class(statement):
     assert token.type == NAME
 
     token = next(tokens)
-    if token.string == "(":
-        token = next(tokens)
-        depth = 1
-        while depth:
-            if token.string == "(":
-                depth += 1
-            if token.string == ")":
-                depth -= 1
+
+    def _optional_nested_brackets(open: str, close: str):
```

Comment:
I've extracted the repeated code here into _optional_nested_brackets.

Reply:
I'm thinking of reverting this refactoring of _optional_nested_brackets and leaving a more complete refactoring of split_class for a latter PR. Agree?

Reply:
Curious to see it.  Go ahead.